yourgov/components/FormMobileFoodVendorPermit/UploadFileTable.tsx

```diff
@@ -0,0 +1,196 @@
+import { forwardRef, Fragment } from 'react';
+import { VisuallyHidden } from '@ag.ds-next/react/a11y';
+import { SkeletonBox, SkeletonText } from '@ag.ds-next/react/skeleton';
+import {
+	Table,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableHeader,
+	TableWrapper,
+	TableRow,
+} from '@ag.ds-next/react/table';
+import { Text } from '@ag.ds-next/react/text';
+import { Button } from '@ag.ds-next/react/button';
+import { Stack } from '@ag.ds-next/react/stack';
+import {
+	AlertFilledIcon,
+	DeleteIcon,
+	UploadIcon,
+} from '@ag.ds-next/react/icon';
+import { Heading } from '@ag.ds-next/react/heading';
+import { StatusBadge } from '@ag.ds-next/react/status-badge';
+import { Document } from './steps/FormStep9';
+
+type UploadFileTableProps = {
+	/** The id of the element to assign the table an accessible name. */
+	ariaLabelledby?: string;
+	/** The id of the heading that describes the table. */
+	headingId?: string;
+	/** The id of the table. */
+	tableId?: string;
+	/** Open drawer. */
+	openDrawer: (document: Document) => void;
+	/** Remove an uploaded file. */
+	removeFile: (document: Document) => void;
+	/** Data for upload documents table. */
+	documents: Document[];
+};
+
+export const UploadFileTable = forwardRef<
+	HTMLTableElement,
+	UploadFileTableProps
+>(function UploadFileTable(
+	{ ariaLabelledby, documents, openDrawer, removeFile, tableId },
+	ref
+) {
+	const error = false;
+	const loading = false;
+
+	if (error) {
+		return (
+			<Stack gap={1}>
+				<AlertFilledIcon color="error" size="lg" />
+				<Heading type="h2" fontSize="lg">
+					Failed to load
+				</Heading>
+				<Text>
+					There was an error loading the data. Please try refreshing the page to
+					try again.
+				</Text>
+			</Stack>
+		);
+	}
+
+	return (
+		<Stack gap={0.5}>
+			<TableWrapper>
+				<Table
+					aria-labelledby={ariaLabelledby}
+					id={tableId}
+					ref={ref}
+					tabIndex={-1}
+					tableLayout="fixed"
+				>
+					<TableHead>
+						<TableRow aria-rowindex={1}>
+							{headers.map(({ label, width }) => {
+								return (
+									<TableHeader key={label} scope="col" width={width}>
+										{label}
+									</TableHeader>
+								);
+							})}
+						</TableRow>
+					</TableHead>
+					<TableBody>
+						{loading ? (
```

Comment:
I don't think this ever gets used?

Reply:
It doesn't but I'd like to leave it and address it later when we look at these tables more holistically.

Reply:
- [ ] Remove